### PR TITLE
Reflect design changes in registreation flow

### PR DESCRIPTION
**Pull request by richtier**
[Designs have changed a lot](https://drive.google.com/drive/folders/0BxjFGIdOidTubUlPZldpa0Z4dFU), so reflecting those changes in the forms:

step one:
screenshot from 2016-10-05 11 33 05

step two:
screenshot from 2016-10-05 11 33 34

step three:
screenshot from 2016-10-05 11 34 05

### CODE REVIEW

**Comment by konradko** [comment]
@richtier conflicts

File: ui/forms.py [hunk 1]
```diff
@@ -55,26 +55,24 @@ class CompanyForm(forms.Form):
...
-    def clean_company_email_confirmed(self):
-        email = self.cleaned_data.get('company_email')
-        email_confirmed = self.cleaned_data.get('company_email_confirmed')
-        if email and email != email_confirmed:
-            raise forms.ValidationError('Your emails must match')
-        return email_confirmed
 
+class AimsForm(forms.Form):
+    AIMS = (
+        ('', ''),
```
Comment (konradko) [inline ui/forms.py]: why allow empty?

File: ui/forms.py [hunk 2]
```diff
@@ -55,26 +55,24 @@ class CompanyForm(forms.Form):
...
 
-    def clean_company_email_confirmed(self):
-        email = self.cleaned_data.get('company_email')
-        email_confirmed = self.cleaned_data.get('company_email_confirmed')
-        if email and email != email_confirmed:
-            raise forms.ValidationError('Your emails must match')
-        return email_confirmed
 
+class AimsForm(forms.Form):
+    AIMS = (
```
Comment (konradko) [inline ui/forms.py]: ee, will that work? Should also be in a separate 'constants' file, as will be reused

File: ui/forms.py [hunk 3]
```diff
@@ -55,26 +55,24 @@ class CompanyForm(forms.Form):
...
-    password = forms.CharField(widget=forms.PasswordInput())
-    password_confirmed = forms.CharField(widget=forms.PasswordInput())
 
-    def clean_password_confirmed(self):
-        password = self.cleaned_data.get('password')
-        password_confirmed = self.cleaned_data.get('password_confirmed')
-        if password and password != password_confirmed:
-            raise forms.ValidationError('Your passwords must match')
-        return password_confirmed
+class PersonalForm(forms.Form):
```
Comment (konradko) [inline ui/forms.py]: ``UserForm``?

File: ui/forms.py (hunk 1 above)
Comment (richtier) [inline ui/forms.py]: User can't really use empty - form validation prevents empty value being used.

The first option in the list is pre-selected by Django when the field renders. If we have an actual answer as the pre-selected value then that default value will be submitted by lazy users who just want to click 'next' without reading the page.

That will then mean we cannot trust data that shows 'answer 1' is more popular than 'answer 2', for example.

File: ui/forms.py (hunk 2 above)
Comment (richtier) [inline ui/forms.py]: yeah it works. shall move though sure.

File: ui/templates/personal-form.html [hunk 4]
```diff
@@ -0,0 +1 @@
+{{form.as_p}}
```
Comment (konradko) [inline ui/templates/personal-form.html]: user-from.html?

Comment (richtier) [inline ui/templates/personal-form.html]: done

**Comment by richtier** [comment]
I've made all the required changes. merging.

**Pull request merged**